.travis.yml

```diff
@@ -25,8 +28,8 @@ env:
         - SYMFONY_DEPRECATIONS_HELPER=weak
 
 before_install:
-    - composer self-update
     - if [[ "$TRAVIS_PHP_VERSION" != "nightly" ]] && [[ "$TRAVIS_PHP_VERSION" != "hhvm" ]]; then phpenv config-rm xdebug.ini; fi;
+    - composer self-update
```

Comment:
Given that Travis has not shipped an environment update since February 2015 (they planned to do it every 2 months, failed for the first update and then never updated again AFAIK), we need this. A 1-year-old composer is too old for the version constraints used on Packagist.